src/plugins/saved_objects_management/public/management_section/object_view/components/header.tsx

```diff
@@ -35,29 +43,10 @@ export const Header = ({
   onDeleteClick,
 }: HeaderProps) => {
   return (
-    <EuiPageContentHeader>
-      <EuiPageContentHeaderSection>
-        <EuiTitle>
-          {canEdit ? (
-            <h1>
-              <FormattedMessage
-                id="savedObjectsManagement.view.editItemTitle"
-                defaultMessage="Edit {title}"
-                values={{ title: type }}
-              />
-            </h1>
-          ) : (
-            <h1>
-              <FormattedMessage
-                id="savedObjectsManagement.view.viewItemTitle"
-                defaultMessage="View {title}"
-                values={{ title: type }}
-              />
-            </h1>
-          )}
-        </EuiTitle>
-      </EuiPageContentHeaderSection>
-      <EuiPageContentHeaderSection>
+    <EuiPageHeader
+      bottomBorder
+      pageTitle={renderConditionalTitle(canEdit, type)}
+      rightSideItems={[
         <EuiFlexGroup responsive={false}>
           {canViewInApp && (
             <EuiFlexItem grow={false}>
```

Comment:
You can actually get rid of this entire flex group and just pass each button in as an item in the array like:

```suggestion
      rightSideItems={[ canViewInApp && <EuiButton>View</EUiButton>, canDelete & <EuiButton>Delete</EUiButton> ]}
```

Reply:
@cchaos I've implemented the changes and reverted the files.